apps/ledger-live-desktop/src/renderer/families/celo/UnlockFlowModal/types.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import type { TFunction } from "react-i18next";
+import type { Device } from "@ledgerhq/live-common/lib/hw/actions/types";
+import type { Step } from "~/renderer/components/Stepper";
+
+import type { Account, TransactionStatus, Operation } from "@ledgerhq/live-common/lib/types";
+
+import type { Transaction } from "@ledgerhq/live-common/lib/families/celo/types";
+
+export type StepId = "amount" | "connectDevice" | "confirmation";
+
+export type StepProps = {
```

Comment:
Then maybe this warrants more discussion with @andyhass. There's a fine line between DRYing the code and breaking architectural conventions set by Ledger.

Reply:
I had the same thought as you @marco-figment, but I decided I was fine with the current code just because it kept things modular.  The code within these Modal directories are isolated and independent, which I think has a lot of value.